ibm/mas_devops/roles/suite_install/README.md

```diff
@@ -24,6 +24,9 @@ Opitional fact, if not provided the role will use the default cluster subdomain
 ### mas_instance_id
 Defines the instance id to be used for MAS installation
 
+### mas_trust_default_cas
+Defines whether Default CAs are included in MAS trust stores. Can be set by the `MAS_TRUST_DEFAULT_CAS` environment variable
```

Comment:
Should state something like "only takes effect with MAS 8.11 or newer"

Reply:
added "This only has an effect in IBM Maximo Application Suite version 8.11 and above."